docs/app/pages/showcase/visualizations/organization-chart/organization-chart.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="chart-container" #container>
```

Comment:
Seems to be some strange styling happening or org chart:

[![Image from Gyazo](https://i.gyazo.com/76b775f4f675d6ac73f65aa2cfe89bd3.gif)](https://gyazo.com/76b775f4f675d6ac73f65aa2cfe89bd3)